src/statd/python/cli_pretty/cli_pretty.py

```diff
@@ -1255,6 +1264,36 @@ def show_dhcp_server(json, stats):
         print(Decore.invert(hdr))
         server.print()
 
+def show_lldp(json):
```

Comment:
According to PEP-8 methods outside of a class should be interleaved with two empty lines. (See above comment as well)